cpp/oneapi/dal/algo/knn/backend/gpu/infer_kernel_brute_force_dpc.cpp

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#define DAAL_SYCL_INTERFACE
+#define DAAL_SYCL_INTERFACE_USM
+
+#include "src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi.h"
+
+#include "oneapi/dal/algo/knn/backend/gpu/infer_kernel.hpp"
+#include "oneapi/dal/algo/knn/backend/model_interop.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/common_dpc.hpp"
+#include "oneapi/dal/backend/interop/error_converter.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+#include "oneapi/dal/detail/common.hpp"
+
+namespace oneapi::dal::knn::backend {
+
+using dal::backend::context_gpu;
+using namespace daal::data_management;
+
+namespace daal_knn = daal::algorithms::bf_knn_classification;
+namespace interop  = dal::backend::interop;
+
+template <typename Float>
+using daal_knn_brute_force_kernel_t =
+    daal_knn::prediction::internal::KNNClassificationPredictKernelUCAPI<Float>;
+
+template <typename Float>
+static infer_result call_daal_kernel(const context_gpu& ctx,
+                                     const descriptor_base& desc,
+                                     const table& data,
+                                     model m) {
+    auto queue = ctx.get_queue();
```

Comment:
See comments https://github.com/oneapi-src/oneDAL/pull/709 form @rlnx 

```suggestion
    auto& queue = ctx.get_queue();
```